scss/input/_layout.scss

```diff
@@ -191,6 +199,122 @@
         }
     }
 
+    // Forms
+    $form-spacer: $padding-x * 2 !default;
+
+    .k-form,
+    .k-form-inline {
+        font-size: $font-size;
+        line-height: $line-height;
+        padding: $form-spacer;
+
+        fieldset {
+            border-width: 1px 0 0;
+            border-style: solid;
+            margin: ($form-spacer * 2) 0;
+            padding: 0;
+
+            &:first-child:first-of-type {
+                margin-top: 0;
+            }
+
+            &:last-child:last-of-type {
+                margin-bottom: 0;
+            }
+
+            legend {
+                font-size: $font-size-sm;
+                font-weight: 600;
+                line-height: 1;
+                margin-bottom: $padding-y-lg * 2;
+                text-transform: uppercase;
+                padding: 0 $padding-x 0 0;
+                width: auto;
+            }
+        }
+
+        .k-form-field {
+            display: block;
+            margin-bottom: $padding-y-lg * 2;
+
+            > span {
+                line-height: $line-height;
+            }
+
+            &:last-child {
+                margin-bottom: 0;
+            }
+        }
+
+        .k-alert-error {
+            font-size: $font-size-sm;
+            margin-top: $padding-y * 2;
+        }
+
+        .k-checkbox-label,
+        .k-radio-label {
+            margin-right: $padding-x * 2;
+        }
+    }
+
+    .k-form {
+        input,
+        label:not(.k-checkbox-label):not(.k-radio-label),
+        .k-widget {
+            display: block;
+        }
+
+        .k-form-field {
+            > span {
+                display: block;
+                padding: $padding-y 0;
+            }
+
+            > input {
+                width: 100%;
+            }
+        }
+
+        .k-field-info {
+            display: inline-block;
+            font-size: ($font-size-xs / $font-size) * 1em;
+            margin: 0 $padding-x;
+        }
+    }
+
+    .k-form-inline {
+        $inline-form-element-width: 25% !default;
+
+        .k-form-field {
+            > span,
+            > input {
+                display: inline-block;
+                box-sizing: border-box;
+            }
+
+            > span,
```

Comment:
Same question as with block forms -- what if my input is combobox? It renders a span.